xapian-letor/include/xapian-letor/ranker.h

```diff
@@ -28,61 +30,116 @@
 #include <xapian/visibility.h>
 
 #include "featurevector.h"
+#include "letor_error.h"
 
 #include <list>
+#include <iostream>
```

Comment:
`<iostream>` shouldn't be needed in the library code (and you've added it to the wrong place in the sorted list too).